pkg/artifactory/resource_artifactory_local_repository_test.go

```diff
@@ -661,3 +662,47 @@ func TestAccLocalAllRepoTypes(t *testing.T) {
 		})
 	}
 }
+
+func makeLocalRepoTestCase(repoType string, t *testing.T) (*testing.T, resource.TestCase) {
+	name := fmt.Sprintf("terraform-local-%s-%d-full", repoType, rand.Int())
+	resourceName := fmt.Sprintf("artifactory_local_%s_repository.%s", repoType, name)
+	repoLayoutRef := getDefaultLocalRepoLayoutRef(repoType, true)
```

Comment:
As mentioned before, please use separate test utility func instead of combining with production code

Reply:
Improved. Function `getDefaultLocalRepoLayoutRef` has been removed and new improvement has been built.